pelita/game.py

```diff
@@ -84,13 +84,17 @@ def controller_exit(state, await_action='play_step'):
 
 def run_game(team_specs, *, max_rounds, layout_dict, layout_name="", seed=None,
              max_team_errors=5, timeout_length=3, viewers=None, controller=None, viewer_options=None,
-             store_output=False):
+             store_output=False, team_names=(None, None), allow_exceptions=False):
```

Comment:
Yes, it is used in https://github.com/ASPP/pelita_template/blob/master/demo10_background_games.py . It should stay a tuple, IMHO. 
It is very useful to pass the names if you do local games, because often the `move` function is just called `move` and comes from a different module. Like in the demo linked above.
If you don't set the names, the output is something like "move (local-team) wins over move (local-team)" :)))
I don't thnk `team_names` will be used for remote clients, given that setting up a remote game with `run_game` is out-of-scope for the students. So that fact the json will be forced to convert in the rare cases this become relevant is not so bad.

Reply:
Sure, I see why it should be used, but I’m betting that whoever uses this feature for testing will eventually call `random.shuffle([move1, move2])` and forget to adapt `team_names` and be even more confused on the outcome. You have my word for it. :)

Reply:
OK, bet accepted!